packages/paste-core/components/spinner/stories/index.stories.tsx

```diff
@@ -178,3 +180,76 @@ export const InFullscreenOverlay: React.ReactNode = () => {
     </Box>
   );
 };
+
+export const CustomizedSpinner = (): React.ReactNode => {
+  const currentTheme = useTheme();
+  return (
+    <CustomizationProvider
+      theme={currentTheme}
+      elements={{
+        SPINNER_ICON_WRAPPER: {
+          color: 'colorTextIcon',
+        },
+        CUSTOM_SPINNER_ICON_WRAPPER_A: {
+          size: 'sizeIcon80',
+          backgroundColor: 'colorBackgroundBrand',
+          color: 'colorTextInverse',
+          alignSelf: 'flex-start',
+          padding: 'space20',
+        },
+        CUSTOM_SPINNER_ICON_WRAPPER_B: {
+          size: 'sizeIcon60',
+          alignSelf: 'flex-end',
+          color: 'colorTextNew',
+        },
+      }}
+    >
+      <Box width="50%">
+        <Card>
+          <Box display="flex" justifyContent="space-between" alignItems="center" padding="space40">
+            <Spinner color="colorText" title="Loading content" decorative={false} />
+
+            {/* Note: color passed here is ignored due to customization rules above */}
+            <Spinner
+              element="CUSTOM_SPINNER_ICON_WRAPPER_A"
+              color="colorText"
+              title="Loading content"
+              decorative={false}
+            />
+            <Spinner color="colorText" title="Loading content" decorative={false} />
+
+            {/* Note: color passed here is ignored due to customization rules above */}
+            <Spinner
+              element="CUSTOM_SPINNER_ICON_WRAPPER_B"
+              color="colorText"
+              title="Loading content"
+              decorative={false}
+            />
+
+            <Spinner color="colorText" title="Loading content" decorative={false} />
+
+            {/* Note: color passed here is ignored due to customization rules above */}
+            <Spinner
```

Comment:
Is this a dupe of line 213?

Reply:
Yeah I was trying to demonstrate customizability between instances, and shared customization. Do you think this should be removed?

Reply:
I went ahead and removed them FYI